spec/factories/in_person_enrollments.rb

```diff
@@ -3,18 +3,34 @@
     user { association :user, :signed_up }
     profile { association :profile, user: user }
     unique_id { Faker::Number.hexadecimal(digits: 18) }
+    current_address_matches_id { true }
+    selected_location_details do { name: 'BALTIMORE' } end
```

Comment:
I'd only expect to see `do; end` with a multi-line block (and surprised it's not in our Rubocop config)

```suggestion
    selected_location_details { { name: 'BALTIMORE' } }
```

Or:

```suggestion
    selected_location_details do
      { name: 'BALTIMORE' }
    end
```

Reply:
It's now in the rubocop config: https://github.com/18F/identity-idp/pull/6875

Reply:
Fixed